processing/sql/dml/013_prg_pa_update.sql

```diff
@@ -0,0 +1,21 @@
+-- okreslenie nazwy ulic
+-- zgodnosc simc, nazwa ulicy (nazwa_1)
+update prg.pa as pa
+set teryt_ulica_2 = trim(concat(cm.m, ' ', (u1.nazwa_2 || ' '), u1.nazwa_1)), teryt_ulic_2 = u1.sym_ul, match_type = 3
```

Comment:
skąd ten match_type ? w innych chyba tego nie było?

Reply:
Pozostałość po debugowaniu. Nie wiem dlaczego akurat tylko w tym zapytaniu to zostało. Poprawione.